### PR TITLE
OLM integration

### LINKED ISSUE #155: OLM support
Do whatever packaging is needed to make it available through the operator hub and have a pleasant user-experience in installing and upgrading.

### PR DESCRIPTION
**Pull request by sabre1041**
OLM integration to resolve #155

### CODE REVIEW

**Comment by sabre1041** [comment]
@davidkarlsen Here is an initial version of what the operator will look like in OLM. 
Screen Shot 2021-01-20 at 1 12 32 AM

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Current Operator version
-VERSION ?= 0.5.0
+VERSION ?= 0.8.1
 # Default bundle image tag
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
+BUNDLE_IMG ?= quay.io/evryfs/github-actions-runner-operator:$(VERSION)
```
Comment (davidkarlsen) [inline Makefile:4]: Wont't this conflict with the actual image for the operator, or at least be confusing - is there any common pattern to follow?

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Current Operator version
-VERSION ?= 0.5.0
```
Comment (davidkarlsen) [inline Makefile:2]: would it make sense to follow the same pattern as for the operator, to use the git-tag?

File: Makefile (hunk 1 above)
Comment (sabre1041) [reply Makefile]: Pushed a change. You can set environment variables to override at runtime

File: Makefile (hunk 2 above)
Comment (sabre1041) [reply Makefile:2]: I reset it to `0.0.1` and will be overridden at runtime

**Comment by sabre1041** [comment]
@davidkarlsen made some updates. going to work on the description portion with better documentation on installation steps. then we should be in a good place

**Comment by davidkarlsen** [comment]
@sabre1041 is this still WIP or ready to be merged now?

**Comment by sabre1041** [comment]
@davidkarlsen want to get #171 merged and then ill update and we will be good to go

**Comment by sabre1041** [comment]
@davidkarlsen should be good now

**Pull request merged**